browser/playlist/test/playlist_download_request_manager_browsertests.cc

```diff
@@ -98,9 +98,13 @@ class PlaylistDownloadRequestManagerBrowserTest : public PlatformBrowserTest {
 
     base::RunLoop run_loop;
     EXPECT_CALL(observer, OnMediaFilesUpdated(testing::_, testing::_))
-        .WillOnce(
+        .Times(testing::AtLeast(1))
```

Comment:
@sangwoo108 What items does `actual_items` contain exactly on each `OnMediaFilesUpdated` call? Are you getting two empty vectors? FWIW, the nondeterministic behavior of the detector script has been restricted on my refactoring branch (https://github.com/brave/brave-core/compare/master...szilard/dont-queue-background-webcontents-requests). Also, since `PlaylistDownloadRequestManager` will be gone soon (well, it's already gone on my branch), and the tests will need to be rewritten for the new component, I'm not sure if it's worth spending too much time fixing this test, rather than just disable it.

Reply:
> What items does actual_items contain exactly on each OnMediaFilesUpdated call

I'm not sure. This patch is based on the CI failure https://github.com/brave/brave-browser/issues/36452

Reply:
> Also, since PlaylistDownloadRequestManager will be gone soon (well, it's already gone on my branch),

Yeah, if your branch will be merged before we release, I'm okay with disabling it.

Reply:
Yes. Raised https://github.com/brave/brave-core/pull/22437, and closing yours. 👍🏻